pkg/common/health/config.go

```diff
@@ -0,0 +1,39 @@
+package health
+
+import "time"
+
+type Config struct {
+	Address  string        `hcl:"listen_address"`
```

Comment:
Should we have a localhost default here? In exchange for using listen address we could add an `enabled` bool

Reply:
My use-case is fine with plaintext being localhost-only, and this will match the prometheus one (just set a port).

Reply:
this is now configurable as discussed above